inference-engine/src/vpu/graph_transformer/src/stages/static_shape_nms.cpp

```diff
@@ -119,6 +149,21 @@ void FrontEnd::parseStaticShapeNMS(const Model& model, const ie::CNNLayerPtr& la
 
     auto stage = model->addNewStage<StaticShapeNMS>(layer->name, StageType::StaticShapeNMS, layer, usedInputs, DataVector{outIndices, outShape});
     stage->attrs().set<bool>("center_point_box", centerPointBox);
+
+    auto spatDim = inputs[0]->desc().dim(Dim::H);
+
+    const int ALIGN_VALUE = 64;
+    const int buffer_size0 = 2 * sizeof(int16_t) * 4 * spatDim;
```

Comment:
Could you please use `camelCase` for all variables names?

Reply:
Fixed